src/Microsoft.AspNetCore.DataProtection/ActivatorExtensions.cs

```diff
@@ -37,55 +37,8 @@ public static T CreateInstance<T>(this IActivator activator, string implementati
         public static IActivator GetActivator(this IServiceProvider serviceProvider)
```

Comment:
This is strange as an extension method. Services are one of those things that you should know if it's null or not.